appconfig/tablekeycols.csv

```diff
@@ -0,0 +1,6 @@
+tablename,keycol
+packets,
+quote,sym
+quote_iex,
+trade,sym
+trade_iex,
```

Comment:
should none of the tables have a keycol? where are `quote` & `trade`? (I could easily be missing something, but no context in PR description etc.)

Reply:
ah apparently I was just being blind and missed quote & trade.

why do the other tables not need a keycol?

Reply:
Those tables are just empty. They shouldn't have any data to save down.

Reply:
I don't think the tables are "meant" to be empty - that probably means the IEX feed isn't working/isn't configured properly in your dev stack etc.

I think we should probably have all the tables configured with sym as keycol by default

Reply:
Ok, will see if I can get those working today. The end of period savedown should default to sym anyway when no value is given, I'll double check that today.